### DISCUSSION TITLE
How to generate contract tests into a folder with a given name

### DISCUSSION THREAD

**Discussion by PaulCluver** [apideck-libraries/portman#492] [Q&A] [answered]
Hi there, 

I am generating some contract tests for an API that I am testing. I would like the contract tests to be generated into a folder named "Contract Tests". Please see below for my currrent generated structure and the ideal generated structure:

**Currrent Generated Structure** 
![image](https://github.com/apideck-libraries/portman/assets/5680263/5a0ec592-418b-494a-a571-9594b996b225)

**Ideal Generated Structure**
![image](https://github.com/apideck-libraries/portman/assets/5680263/e6124398-5709-495c-b290-b151a6b1be4d)

If anyone could provide me with some advice as to how I could achieve the desired result it would be most appreciated. Please find attached images of my portman-config.yaml and test-contracts.yaml as reference. 

Kind regards,
 
![image](https://github.com/apideck-libraries/portman/assets/5680263/06983b14-2f07-4289-8562-cc31dd94c23d)
![image](https://github.com/apideck-libraries/portman/assets/5680263/c198217f-b641-4504-8a33-81836c74ad32)

**Comment by thim81** [comment, marked as answer]
hi @PaulCluver 

There is a CLI parameter `--bundleContractTests` that would just that.
See the example with the before and after screenshots: https://github.com/apideck-libraries/portman/tree/main/examples/testsuite-contract-tests

**Reply by PaulCluver** [reply]
Thank you so much. It worked perfectly.
